includes/class-wc-payment-gateway-wcpay.php

```diff
@@ -265,6 +265,10 @@ public function is_in_test_mode() {
 	 * @return bool Whether the gateway is enabled and ready to accept payments.
 	 */
 	public function is_available() {
+		if ( 'USD' !== get_woocommerce_currency() ) {
```

Comment:
Can we do something like `if ( ! $this->is_in_dev_mode() && 'USD' !== get_woocommerce_currency() )` here, so we were able to keep working on currencies related topics, while production keeps working with USD only?

Reply:
Or `if ( (! defined( 'WCPAY_DEV_MODE' ) || ! WCPAY_DEV_MODE) && 'USD' !== get_woocommerce_currency() )`, for staging systems to also operate with USD only.

Reply:
Or
```php
$is_extension_dev_mode = defined( 'WCPAY_DEV_MODE' ) && WCPAY_DEV_MODE;
if ( ! $is_extension_dev_mode && 'USD' !== get_woocommerce_currency() ) 
```

Reply:
@kalessil this is a great suggestion, but since we're in crunch mode trying to get this out, would you mind opening a PR for this? We could merge it after the release today.